Xiaomi/Redmi7onc/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.xiaomi.redmi7onc"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+                android:requiredSystemPropertyValue="+*iaomi/onc_*"
```

Comment:
`ro.vendor.build.fingerprint=xiaomi/onc_eea/onc:9/PKQ1.181021.001/V10.3.4.0.PFLEUXM:user/release-keys`
`ro.vendor.build.fingerprint=xiaomi/onc/onc:9/PKQ1.181021.001/V10.3.3.0.PFFINXM:user/release-keys`
`ro.vendor.build.fingerprint=xiaomi/onc/onc:9/PKQ1.181021.001/V10.3.4.0.PFLCNXM:user/release-keys`

`onc*` is needed to support all variants